docs/walkthrough.md

```diff
@@ -32,6 +32,18 @@ be done with Helm setup.
 If you don't already have Helm v2, see the
 [installation instructions](https://github.com/kubernetes/helm/blob/master/docs/install.md).
 
+### RBAC Considerations
+
+If your kubernetes cluster has [RBAC](https://kubernetes.io/docs/admin/authorization/rbac/)
+enabled, you must ensure that the default service account for the `kube-system`
+namespace has the `cluster-admin` role:
+
+```console
+kubectl create clusterrolebinding default-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default
```

Comment:
kubeadm sets up this level of access, and while I have to include the disclaimer that kubeadm is not production ready, I have to somewhat assume that a decision was made that this is the correct level of access. 

An additional data point is that Bluemix does include this serviceAccount bound to this level of access.